easybuild/easyconfigs/u/UCX/UCX-1.10.0-GCCcore-10.3.0.eb

```diff
@@ -28,6 +32,7 @@ builddependencies = [
 osdependencies = [OS_PKG_IBVERBS_DEV]
 
 dependencies = [
+    ('zlib', '1.2.11'),
```

Comment:
@Micket Any comment on adding this dependency, which doesn't seem to be related to the main purpose of the PR?

Reply:
I just noticed we have missed it adding it. `ldd  $EBROOTUCX/lib/libucs.so`